tests/kafkatest/services/verifiable_producer.py

```diff
@@ -37,7 +39,19 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
 
+        # If security protocol is not specified, use the default in the template properties.
+        self.prop_file = self.render('verifiable_producer.properties')
+        self.security_config = SecurityConfig(security_protocol, self.prop_file)
```

Comment:
@rajinisivaram Actually @granders and I are on the same page with you about using template producer property files to customize testing parameters moving forward. It is just that since we do not use it anywhere yet, adding a dummy empty file seems less attracting for now. So what we are suggesting is that:

a. Moving forward, we should 1) add an non-empty default property file, 2) always require a property file in verifiable_producer.py's config constrcution APIs, and add another API using default file if it is not specified.

b. For now, just keep the API that does not use any property files yet to override the property values, and also get rid of the dummy files.

Does this sound reasonable to you?